### DISCUSSION TITLE
车运动时和静止时被鸟拉屎的概率

### DISCUSSION THREAD

**Discussion by caibingcheng** [caibingcheng/comment#115] [Announcements]
https://bbing.com.cn/202204/birdshit-rate/

**Comment by goodpaperman** [comment]
这是分析了个啥 ：）
重要的不是鸟拉屎的概率，而是拉到车上怎么清理。。

**Reply by caibingcheng** [reply]
是想知道什么情况下，车被鸟屎击中的概率更高，对行人同理。。。但是没啥用，缺少真实数据支撑。
